synapse/lib/stormlib/aha.py

```diff
@@ -85,6 +111,73 @@ async def _methAhaGet(self, svcname, filters=None):
         proxy = await self.runt.snap.core.reqAhaProxy()
         return await proxy.getAhaSvc(svcname, filters=filters)
 
+    async def _methCallPeerApi(self, svcname, apiname, timeout=None, skiprun=None):
+        '''
+        Call an API on an AHA service.
+
+        Args:
+            svcname (str): The name of the AHA service to call
+            apiname (str, tuple): The API name or todo tuple (name, args, kwargs)
+            timeout (int): Optional timeout in seconds
+            skiprun (bool): Optional flag to skip run checks
```

Comment:
This seems to be documented incorrectly. The `skiprun` argument to the AHA APIs allows the caller to specify an instance ( run id ) to skip when enumerating service APIs, under the assumption that the `skiprun` ID matches the caller ID and the caller intends to provide the answer for itself. ( ie, no need to call a thing that calls you back to return the answer to yourself ;) )  If so, it is either a string or None rather than a bool.

Reply:
Thanks for clarification, I wasn't really sure the use-case and this helps.

Reply:
:+1: Anytime!

Reply:
I've updated the `type` and `desc` for `skiprun` let me know if you want any changes.